Cargo.toml

```diff
@@ -9,5 +9,8 @@ edition = "2021"
 yultsur = { git = "https://github.com/leonardoalt/yultsur", branch = "main" }
 num-bigint = "^0.4.3"
 tempfile = "3"
-clap = "^3.2"
+clap = { version = "^3.2", features = ["derive"] }
 num-traits = "^0.2.15"
+
+# misc
+eyre = "0.6"
```

Comment:
can you expand on why this extra dependency is needed?

Reply:
I was experimenting with it but it's not needed for now, just deleted!